.gitignore

```diff
@@ -119,6 +119,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.vscode/
```

Comment:
Nitpick but can you move this down below the `#pycharm` comment, with a comment `vscode`?